src/sider/kort/02-behov/index.tsx

```diff
@@ -78,7 +78,33 @@ const Behov = (): React.JSX.Element => {
         toggle,
     } = useKategorier(!!forsorgerplikt?.harForsorgerplikt, setValue, getValues);
 
+    const {setProcessedData} = useProcessedData();
+
     const onSubmit = (formValues: FormValues) => {
+        const selectedKategorier: string[] = reducer
+            .filter((category) => category.selected)
+            .map((category) => {
+                if (category.text === "Nødhjelp" && category.subCategories?.length) {
+                    const selectedSubCategories = category.subCategories
+                        .filter((subCategory) => subCategory.selected)
+                        .map((subCategory) => subCategory.text);
+
+                    if (selectedSubCategories.length > 0) {
+                        return `${category.text}: ${JSON.stringify(selectedSubCategories)}`;
+                    }
+                }
+                if (category.text === "Annet" && formValues.hvaSokesOm) {
```

Comment:
er ikke denne litt overflødig?

Reply:
Å, ja det er overflødig. Grunnen for den if var at om søker velger "Annet" og om de skriver noe så skal skulle det bli tatt vare på. Så det blir litt feilt å ha return category.text; helt på slutten.

Reply:
Må ta det tilbake det jeg sa om overflødig, uten return category.text; så blir det eneste som blir sendt til amplitude bare Nødhjelp og/eller Annet.